packages/element-templates-json-schema-shared/src/defs/condition.json

```diff
@@ -9,19 +9,21 @@
         "type": {
           "$id": "#/condition/type",
           "const": "simple",
-          "description": "The type of the condition.",
+          "description": "Defines the input type in the properties panel. Can be String, Text, Number, Boolean (checkbox), Dropdown, or Hidden.",
           "default": "simple"
```

Comment:
This defaults to simple, but lists in the description many other types. Is the description matching the actual property described?